.gitlab/package_build/linux.yml

```diff
@@ -81,7 +84,37 @@
   before_script:
     - export RELEASE_VERSION=$RELEASE_VERSION_7
 
-# build Agent 6 binaries for x86_64
+# Temporary custom agent build test to prevent regression
+# This test will be removed when custom path are used to build macos agent
+# with in-house macos runner builds.
+datadog-agent-7-x64-custom-path-test:
+  extends: [.agent_build_x86, .agent_7_build]
+  rules:
+    - !reference [.except_mergequeue]
+    - when: on_success
+  stage: package_build
+  script:
+    - mkdir /custom
+    - export CONFIG_DIR="/custom"
+    - export INSTALL_DIR="/custom/datadog-agent"
+    - !reference [.agent_build_script]
+    - ls -la $OMNIBUS_PACKAGE_DIR
+    - ls -la $INSTALL_DIR
+    - ls -la /custom/etc
+    - (ls -la /opt/datadog-agent 2>/dev/null && exit 1) || echo "/opt/datadog-agent has correctly not been generated"
+    - (ls -la /etc/datadog-agent 2>/dev/null && exit 1) || echo "/etc/datadog-agent has correctly not been generated"
```

Comment:
I suppose one of those was meant to check for `/custom/datadog-agent` instead?

Reply:
Nope checking for /opt and /etc 😄

Reply:
Oh indeed, I didn't see the difference :facepalm: